src/capitalsFirst.js

```diff
@@ -21,7 +21,18 @@
  * @returns {string} - string with uppercase words in front
  */
 function capitalsFirst(str) {
-  // write code here
+  const wordsArray = str.split(' ');
+  function checkUpper(word) {
```

Comment:
1. checkWord is too generic. Not clear what function is doing. 
2. move it outside of capitalsFirst method.

Reply:
3. if you do it as a lambda expression you can keep it here.

Reply:
3. If i do it as lambda, lint returns error due to string overlength.
2. I left it here because of //write code here
1. Looking for balance between 'not too generic' & its length. May be, it is better to add second flag-argument to switch between lower and upper cases and name it 'checkFirstLetter'?

Reply:
3. that shouldn't be a problem. you can put linebreaks in lambda function and make it take 2 lines.
1. I don't think that's a good idea.